manifests/noderesourcetopology/scheduler-config.yaml

```diff
@@ -2,9 +2,6 @@ apiVersion: kubescheduler.config.k8s.io/v1beta1
 kind: KubeSchedulerConfiguration
 leaderElection:
   leaderElect: false
-clientConnection:
-  # kubeconfig: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
-  kubeconfig: "/etc/kubernetes/scheduler.conf"
```

Comment:
Nope, this is still needed.

Reply:
Hmm, okay. I didn't realize that. Thanks for letting me know.